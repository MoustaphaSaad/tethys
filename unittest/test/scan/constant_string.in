constant msg "Hello, World!\n\0"