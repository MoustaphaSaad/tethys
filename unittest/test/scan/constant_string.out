line: 1, col: 0, kind: "CONSTANT" str: "constant"
line: 1, col: 9, kind: "<ID>" str: "msg"
line: 1, col: 13, kind: "<STRING>" str: "Hello, World!\n\0"
